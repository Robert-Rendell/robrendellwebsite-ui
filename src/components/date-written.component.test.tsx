import React from "react";
import { render } from "@testing-library/react";
import { DateWritten } from "./date-written.component";

describe("DateWrittenComponent", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<DateWritten/>);
    expect(asFragment()).toMatchSnapshot();
  });
});