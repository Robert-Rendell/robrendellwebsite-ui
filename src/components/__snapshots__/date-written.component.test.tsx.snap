// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateWrittenComponent should match snapshot 1`] = `
<DocumentFragment>
  <hr />
  <span>
    [
    <i>
      Date Written: 
    </i>
    ]
  </span>
</DocumentFragment>
`;
